src/client/pkg/uuid/uuid_test.go

```diff
@@ -0,0 +1,35 @@
+package uuid
+
+import (
+	"testing"
+)
+
+func TestIsUUIDWithoutDashes(t *testing.T) {
+	if len(NewWithoutDashes()) != 32 {
+		t.Error()
+	}
+
+	if !IsUUIDWithoutDashes("09abcd098faa4fd98643023485739adb") {
+		t.Error()
+	}
+
+	// 13 character is 4
+	if IsUUIDWithoutDashes("09abcd098faaefd98643023485739adb") {
+		t.Fail()
+	}
+
+	// Length 32
+	if IsUUIDWithoutDashes("09abcd098faaefd98643023485739adbabc") {
+		t.Fail()
+	}
+
+	// Hexadecimal
+	if IsUUIDWithoutDashes("09abcd098faa4fd98643023485739xyz") {
+		t.Fail()
+	}
+
+	// Generated
+	if !IsUUIDWithoutDashes(NewWithoutDashes()) {
+		t.Fail()
+	}
+}
```

Comment:
Check out `src/client/pkg/require` it has some helper functions which would make these all 1 liners.
They'll also print out a bit more information when the test does fail.